docs/readmes/howtos/README_package.md

```diff
@@ -51,7 +48,7 @@ you had released your package and run `apt-get install magma`.
 
 If you want to test upgrading versions, skip the part where you spin up a fresh
 prod VM, and instead set up a VM with whatever version you're trying to test
-upgrade from. For example, if you want to test upgrading from the currrently
```

Comment:
![](https://media2.giphy.com/media/bC9czlgCMtw4cj8RgH/giphy.gif)

Reply:
I considered changing it to `currrrently`